resources/templates/emails/pmc-report.ftl

```diff
@@ -96,30 +96,79 @@
-     also provides more insight into the activity of the Program's Projects and Working Groups.</p>
+  <h3>${program.program_short_name} PMC (Program Management Committee) Report as of ${now}</h3>
+  <p>The latest <a href="https://metrics.finos.org/app/kibana?#/dashboard/0e542930-4f2d-11e9-9e7c-eb1eab055f1f?_g=(refreshInterval:(display:Off,pause:!f,value:0),time:(from:now-6m,mode:quick,to:now))&_a=(filters:!(),query:(query_string:(analyze_wildcard:!t,query:'cm_program:%22${program.program_short_name}%22')))">${program.program_name} Program's metrics dashboard</a> is now available in <a href="https://metrics.finos.org/app/kibana?#/dashboard/0e542930-4f2d-11e9-9e7c-eb1eab055f1f?_g=(refreshInterval:(display:Off,pause:!f,value:0),time:(from:now-6m,mode:quick,to:now))&_a=(filters:!(),query:(query_string:(analyze_wildcard:!t,query:'cm_program:%22${program.program_short_name}%22')))">metrics.finos.org</a>. Your program <a href="https://metrics.finos.org/app/kibana?#/dashboard/0e542930-4f2d-11e9-9e7c-eb1eab055f1f?_g=(refreshInterval:(display:Off,pause:!f,value:0),time:(from:now-6m,mode:quick,to:now))&_a=(filters:!(),query:(query_string:(analyze_wildcard:!t,query:'cm_program:%22${program.program_short_name}%22')))">metrics dashboard</a> includes <a href="https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/93225748/Board+Reporting+and+Program+Health+Checks">measures used to track FINOS program health</a> that presently can be generated and calculated by data available to FINOS.</p>
+
+  <p>Please note that some metrics required for quarterly board reports must be tracked and calculated by PMCs themselves. For more information see the “Where to Find Program Measures and Data“ section <a href="https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/93225748/Board+Reporting+and+Program+Health+Checks">in the Board Reporting and Program Health Check page in the FINOS Community Handbook</a>.</p>
+
+  <p>In addition to the data available in the <a href="https://metrics.finos.org/app/kibana?#/dashboard/0e542930-4f2d-11e9-9e7c-eb1eab055f1f?_g=(refreshInterval:(display:Off,pause:!f,value:0),time:(from:now-6m,mode:quick,to:now))&_a=(filters:!(),query:(query_string:(analyze_wildcard:!t,query:'cm_program:%22${program.program_short_name}%22')))">metrics dashboard</a>, please also review both the basic data and program steering warnings and measures below. For more information on how to interpret and follow-up on the steering data, see the <a href="https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/118292491/PMC+Monthly+Reports">PMC Monthly Reports page</a> of the FINOS Community Handbook, which includes recommended remedies and interventions.</p>
+
+  <hr/>
+  <h2>${program.program_name} Program Master Data</h2>
+  <h3>PMC Lead</h3>
+  <ul><li>${pmc_lead}</li></ul>
```

Comment:
This will fail if the PMC lead is null (which, at least last time I had access to this data, was possible).

I would instead recommend something like: ```${pmc_lead!"Position currently unfilled"}```